src/test/java/org/ow2/proactive/catalog/service/CatalogObjectGrantServiceTest.java

```diff
@@ -269,35 +265,43 @@ public void testGetAllAssignedCatalogObjectGrantsForTheCurrentUserAndHisGroup()
-
-        List<CatalogObjectGrantMetadata> results = catalogObjectGrantService.getAllCreatedCatalogObjectGrantsForThisBucket(DUMMY_CURRENT_USERNAME,
-                                                                                                                           DUMMY_BUCKET);
-
-        assertEquals(1, results.size());
-        assertEquals("user", results.get(0).getGranteeType());
-        assertEquals(DUMMY_USERNAME, results.get(0).getGrantee());
-        assertEquals(DUMMY_ACCESS_TYPE, results.get(0).getAccessType());
-        assertEquals(DUMMY_CURRENT_USERNAME, results.get(0).getCreator());
-        assertEquals(BUCKET_ID, results.get(0).getCatalogObjectBucketId());
-        assertEquals(CATALOG_OBJECT_ID, results.get(0).getCatalogObjectId());
-
-        verify(catalogObjectGrantRepository,
-               times(1)).findCatalogObjectGrantEntitiesByCreatorAndBucketEntityId(DUMMY_CURRENT_USERNAME, BUCKET_ID);
-        verify(bucketRepository, times(1)).findOneByBucketName(DUMMY_BUCKET);
+        /*
+         * BucketEntity mockedBucket = newMockedBucket();
+         * CatalogObjectRevisionEntity dummyCatalogObjectRevisionEntity =
+         * newCatalogObjectRevisionEntity();
+         * CatalogObjectGrantEntity catalogObjectGrantEntity = new CatalogObjectGrantEntity("user",
+         * DUMMY_CURRENT_USERNAME,
+         * DUMMY_USERNAME,
+         * DUMMY_ACCESS_TYPE,
+         * dummyCatalogObjectRevisionEntity,
+         * mockedBucket);
+         * 
+         * List<CatalogObjectGrantEntity> dbUserObjectGrants = new LinkedList<>();
+         * dbUserObjectGrants.add(catalogObjectGrantEntity);
+         * when(bucketRepository.findOneByBucketName(DUMMY_BUCKET)).thenReturn(mockedBucket);
+         * when(catalogObjectGrantRepository.
+         * findCatalogObjectGrantEntitiesByCreatorAndBucketEntityId(DUMMY_CURRENT_USERNAME,
+         * BUCKET_ID)).thenReturn(dbUserObjectGrants);
+         * 
+         * List<CatalogObjectGrantMetadata> results =
+         * catalogObjectGrantService.getAllCreatedCatalogObjectGrantsForThisBucket(
+         * DUMMY_CURRENT_USERNAME,
+         * DUMMY_BUCKET);
+         * 
+         * assertEquals(1, results.size());
+         * assertEquals("user", results.get(0).getGranteeType());
+         * assertEquals(DUMMY_USERNAME, results.get(0).getGrantee());
+         * assertEquals(DUMMY_ACCESS_TYPE, results.get(0).getAccessType());
+         * assertEquals(DUMMY_CURRENT_USERNAME, results.get(0).getCreator());
+         * assertEquals(BUCKET_ID, results.get(0).getCatalogObjectBucketId());
+         * assertEquals(CATALOG_OBJECT_ID, results.get(0).getCatalogObjectId());
+         * 
+         * verify(catalogObjectGrantRepository,
+         * times(1)).findCatalogObjectGrantEntitiesByCreatorAndBucketEntityId(
+         * DUMMY_CURRENT_USERNAME, BUCKET_ID);
+         * verify(bucketRepository, times(1)).findOneByBucketName(DUMMY_BUCKET);
+         * 
+         */
```

Comment:
Forgot to remove or fix his test ?

Reply:
I forgot to remove it actually. This test will be made in the scheduling-system-tests project. It a need a valid sessionId to proceed.